controllers/usersignup/usersignup_controller_test.go

```diff
@@ -2047,6 +2047,70 @@ func TestUserSignupWithExistingMURDifferentUserIDOK(t *testing.T) {
 	})
 }
 
+func TestUserSignupPropagatedClaimsSynchronizedToMURWhenModified(t *testing.T) {
```

Comment:
And the same question here.

Reply:
We have a huge number of UserSignup unit tests, I did go through them but was unable to identify anything close to what I wanted to test with this new functionality.

Reply:
you updated the `migrateOrFixMurIfNecessary` function in `masteruserrecord.go` file, there is also `masteruserrecord_test.go` file that contains unit-tests for it, one of them is `TestMigrateMurIfNecessary` which would be cool to update as well when you changed the logic of the function

Reply:
btw, the `usersignup_controller_test.go` file has a test called `TestMigrateMur` - to find it was as easy as looking for "migrate" word in the file